doc/interfaces/basic_flow.rst

```diff
@@ -1,42 +1,248 @@
+
+We can use ``dipy_median_otsu`` to build a brain mask for the diffusion data::
+
+    dipy_median_otsu dwi.nii --median_radius 2 --numpass 1 --vol_idx 0 --out_dir out_work
+
+Visualize the mask using ``dipy_horizon``::
+
+    dipy_horizon out_work/brain_mask.nii.gz
+
+.. figure:: https://github.com/dipy/dipy_data/blob/master/cfin_basic2.png?raw=true
+    :width: 70 %
+    :alt: alternate text
+    :align: center
+
+    Visualization of a slice from the generated brain mask
+
+
+Perform DTI using ``dipy_fit_dti`. The input of this function is the DWI data, b-values and b-vector files and the brain mask that we calculated in the previous step::
+
+    dipy_fit_dti dwi.nii dwi.bval dwi.bvec out_work/brain_mask.nii.gz --out_dir out_work/
+
+The default options of the script generate the following files ad.nii.gz, evecs.nii.gz, md.nii.gz,
+rgb.nii.gz, fa.nii.gz, mode.nii.gz, tensors.nii.gz, evals.nii.gz, ga.nii.gz and rd.nii.gz.
+
+Visualize DEC map::
+
+    dipy_horizon out_work/rgb.nii.gz
+
+.. figure:: https://github.com/dipy/dipy_data/blob/master/cfin_basic3.png?raw=true
+    :width: 70 %
+    :alt: alternate text
+    :align: center
+
+    Visualization of a slice from the first volume of DEC image
+
+We can now move to more advanced reconstruction models. One of the fastest we can use is Constant Solid Angle (CSA) ::
+
+    dipy_fit_csa dwi.nii dwi.bval dwi.bvec out_work/brain_mask.nii.gz --out_dir out_work/
+
+Now, to move into doing some tracking we will need some seeds. We can generate seeds in the following way ::
 
-    dipy_fetch cfin_multib
+    dipy_mask out_work/fa.nii.gz 0.4 --out_dir out_work/ --out_mask seed_mask.nii.gz
 
-Create brain mask::
+Create tracks using peaks::
 
-    dipy_median_otsu dwi.nii.gz --out_dir out_work/
+    dipy_track_local out_work/peaks.pam5 out_work/fa.nii.gz out_work/seed_mask.nii.gz --out_dir out_work/ --out_tractogram tracks_from_peaks.trk
 
-Create stopping criteria::
+.. figure:: https://github.com/dipy/dipy_data/blob/master/some_tracks.png?raw=true
+    :width: 70 %
+    :alt: alternate text
+    :align: center
 
-    dipy_fit_dti dwi.nii.gz dwi.bval dwi.bvec out_work/brain_mask.nii.gz --out_dir out_work/
+    Showing tracks from the specific dataset. This dataset contains only a few slices.
 
-Create peaks::
+We can visualize the result using ``dipy_horizon``. The ``--cluster`` option allows to directly see the clusters of the tractogram::
 
-    dipy_fit_csd dwi.nii.gz dwi.bval dwi.bvec out_work/brain_mask.nii.gz --out_dir out_work/
+    dipy_horizon out_work/tracts_from_peaks.trk --cluster
 
-Create seeding mask (this creates seed_mask in new folder)::
+Alternatively, we can create deterministic tracks using the maximum value of a spherical harmonics cone::
 
-    dipy_mask out_work/fa.nii.gz 0.4 --out_dir out_work/ --out_mask seed_mask.nii.gz
+    dipy_track_local peaks.pam5 fa.nii.gz seed_mask.nii.gz --out_tractogram 'tracks_from_sh.trk' --use_sh
```

Comment:
`--use_sh` parameter does not exist anymore. You should use `--tracking_method` and choose your method between:
- "eudx" 
- "deterministic" or "det" for a deterministic tracking
- "probabilistic" or "prob" for a Probabilistic tracking
- "closestpeaks" or "cp" for a ClosestPeaks tracking

Furthermore, `peaks.pam5` is not on the current folder, so you need to do `out_work/peaks.pam5`. The correct command line is:
`dipy_track_local out_work/peaks.pam5 out_work/fa.nii.gz out_work/seed_mask.nii.gz --out_tractogram 'tracks_from_sh.trk' --tracking_method eudx`